docs/faq/index.md

```diff
@@ -17,6 +17,8 @@ and full working examples, visit the [tutorials section](../tutorials/index.md).
 ## Scale
 * [How can I train with multiple CPU/GPUs with data parallelism?](http://mxnet.io/how_to/multi_devices.html)
 
+* [How can I train using multiple machines?](http://mxnet.io/how_to/distributed_training.html)
```

Comment:
Explain how this is different from the previous and next bullet options? What about updating them to be more specific (LSTM example) and update yours to be specific in some way. Also, since they seem to balance off of each other - one being model parallelism and the other being data parallelism, should this now one be right in the middle?

Reply:
Changed this to 
1. train with multiple CPUs/GPUs on a single machine with data parallelism
2. train using multiple machines with data parallelism
3. ...model parallelism

Reply:
Also moved it to faq